src/Bicep.LangServer.IntegrationTests/SnippetTemplatesTests.cs

```diff
@@ -63,6 +65,21 @@ public void VerifySnippetTemplatesAreErrorFree(CompletionData completionData)
             }
         }
 
+        [DataTestMethod]
+        [DynamicData(nameof(GetSnippetCompletionData), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(CompletionData), DynamicDataDisplayName = nameof(CompletionData.GetDisplayName))]
+        [TestCategory(BaselineHelper.BaselineTestCategory)]
+        public void VerifySnippetTemplatesDoNotContainTargetScope(CompletionData completionData)
```

Comment:
No code changes in this PR. SnippetsProvider currently only supports declarations:
https://github.com/Azure/bicep/blob/bdda2d0bc6742ba3792a4628f042713c0d746b8b/src/Bicep.LangServer/Snippets/SnippetsProvider.cs#L114

I had an offline conversation with @MarcusFelling this morning and we want to stick to just declarations for now. Let me know if there are any concerns/questions.